oak_dice/src/evidence.rs

```diff
@@ -32,6 +32,9 @@ const AMD_SEV_SNP_ATTESTATION_REPORT_SIZE: usize = 1184;
 
 static_assertions::const_assert!(REPORT_SIZE >= AMD_SEV_SNP_ATTESTATION_REPORT_SIZE);
 
+/// The size for compound device identifiers.
```

Comment:
Is there a reference for this?

Reply:
Added link to the Open Profile for DICE.